apps/example/.env.example

```
@@ -0,0 +1,6 @@
+# API key from the Payouts section of the Wompi dashboard.
+WOMPI_PAYOUTS_API_KEY=
+# User principal ID paired with the Payouts API key.
+WOMPI_PAYOUTS_USER_PRINCIPAL_ID=
+# Events key used to verify Payouts webhook signatures.
+WOMPI_PAYOUTS_EVENTS_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WOMPI_PAYOUTS_EVENTS_KEY key should go before the WOMPI_PAYOUTS_USER_PRINCIPAL_ID key

(UnorderedKey)